scss/detail.scss

```diff
@@ -0,0 +1,193 @@
+@import 'variables';
+
+.body{
+  font-family: 'Roboto', sans-serif;
+}
+
+.master-container{
+  display: flex;
+  height: 768px;
+  width: 1366px;
+  background-color: $wild-sand;
+  justify-content: center;
+  align-items: center;
+
+  .return-button{
+    display: flex;
+    flex-direction: row;
+    color: $black;
+    font-size: 20px;
+    line-height: 24px;
+    position: absolute;
+    top: 46px;
```

Comment:
Remember that you can set `position: relative` on a parent of this element to simplify the positioning values :wink: